doc/classes/Node.xml

```diff
@@ -122,6 +122,12 @@
 			<description>
 				Adds a child node. Nodes can have any number of children, but every child must have a unique name. Child nodes are automatically deleted when the parent node is deleted, so an entire scene can be removed by deleting its topmost node.
 				If [code]legible_unique_name[/code] is [code]true[/code], the child node will have an human-readable name based on the name of the node being instanced instead of its type.
+				[b]Note:[/b] If the child node already has a parent, the function will fail. Use [method remove_child] first to remove the node from its current parent. For example:
+				[codeblock]
+				if childNode.get_parent():
```

Comment:
GDScript coding styles favors `snake_case` for identifiers, so it would be better to use `child_node` here instead of camelCase.

Apart from that it looks good.

Reply:
Got it!